src/client/game/game.js

```diff
@@ -3,33 +3,79 @@ import { connect } from 'react-redux';
 
 import AnswerInput from './answer-input';
 
-import { setLeader } from './game-actions';
+import { setLeader, getConsonantDispatch, getVowelDispatch, 
+        disableConsonantDispatch, disableVowelDispatch } from './game-actions';
 
-import { setLeaderDispatch } from '../services/cloak-service';
+import { setLeaderDispatch, setConsonantDispatch, messageGetConsonant, 
```

Comment:
This style is ugly AF for long lists. I propose something like this for 3+ object members
```
import {
    setLeader,
    getConsonantDispatch,
    getVowelDispatch,
    disableConsonantDispatch,
    disableVowelDispatch
} from './game-actions';
```